src/graph/AdminJobExecutor.cpp

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "http/HttpClient.h"
+#include "graph/AdminJobExecutor.h"
+#include "process/ProcessUtils.h"
+#include "webservice/Common.h"
+
+#include <folly/executors/Async.h>
+#include <folly/futures/Future.h>
+
+namespace nebula {
+namespace graph {
+
+AdminJobExecutor::AdminJobExecutor(Sentence *sentence,
+        ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<AdminSentence*>(sentence);
+}
+
+void AdminJobExecutor::execute() {
+    LOG(INFO) << __func__ << " enter";
+    auto opEnum = toAdminJobOp(sentence_->getType());
+
+    if (opNeedsSpace(opEnum)) {
```

Comment:
Only call opNeedsSpace here，So , could we if (opEnum == nebula::meta::cpp2::AdminJobOp::ADD) direct？

Reply:
currently, yes. But there will be more job types, job manager will support . That's why I think there should be a function

Reply:
Got